examples/unet_segmentation_3d_array.py

```diff
@@ -148,7 +148,7 @@ def log_training_loss(engine):
 
 # Add stats event handler to print validation stats via evaluator
 logging.basicConfig(stream=sys.stdout, level=logging.INFO)
-val_stats_handler = StatsHandler()
+val_stats_handler = StatsHandler(output_transform=lambda output: (None, None))
```

Comment:
this is not very intuitive, is the intention blocking iteration default output here?

Reply:
fixed in the latest patch.
Thanks.